tasks/tests/test_get_task_comments.py

```diff
@@ -34,20 +34,22 @@ def task(self, user_and_manager):
     def comments(self, user_and_manager, task):
         user, manager = user_and_manager
         comments = {}
-        for i in range(1, NUM_COMMENTS + 1):
-            comments[f'comment{i}'] = Comment.objects.create(appUser=user,
+        for i in range(1, 2 * NUM_COMMENTS_PER_USER + 1):
+            comment_user = user if i <= NUM_COMMENTS_PER_USER else manager
+            comments[f'comment{i}'] = Comment.objects.create(appUser=comment_user,
```

Comment:
maybe instead of all this complication for loop, you can do it with parametrize. 
one time with a user and one time with a manager

Reply:
Done.